Kitodo/src/main/resources/db/migration/V1_1__Add_test_data_to_db.sql

```diff
@@ -0,0 +1,120 @@
+(4, 1),
+(5, 1),
+(6, 1);
+
+INSERT INTO `projectfilegroups` (`ProjectFileGroupID`, `name`, `path`, `mimetype`, `suffix`, `ProjekteID`, `folder`, `previewImage`) VALUES
+(1, 'MAX', 'http://www.example.com/content/$(meta.CatalogIDDigital)/jpgs/max/', 'image/jpeg', 'jpg', 1, '', 0),
+(2, 'DEFAULT', 'http://www.example.com/content/$(meta.CatalogIDDigital)/jpgs/default/', 'image/jpeg', 'jpg', 1, '', 0),
+(3, 'THUMBS', 'http://www.example.com/content/$(meta.CatalogIDDigital)/jpgs/thumbs/', 'image/jpeg', 'jpg', 1, '', 0),
+(4, 'FULLTEXT', 'http://www.example.com/content/$(meta.CatalogIDDigital)/ocr/alto/', 'text/xml', 'xml', 1, '', 0),
+(5, 'DOWNLOAD', 'http://www.example.com/content/$(meta.CatalogIDDigital)/pdf/', 'application/pdf', 'pdf', 1, '', 0);
+
+INSERT INTO `prozesse` (`ProzesseID`, `Titel`, `ausgabename`, `IstTemplate`, `swappedOut`, `inAuswahllisteAnzeigen`, `sortHelperStatus`, `sortHelperImages`, `sortHelperArticles`, `erstellungsdatum`, `ProjekteID`, `MetadatenKonfigurationID`, `sortHelperDocstructs`, `sortHelperMetadata`, `wikifield`, `docketID`) VALUES
+(1, 'Example_Workflow', NULL, b'1', b'0', b'0', '000014086', 0, 0, '2016-10-01 16:49:48', 1, 1, 0, 0, NULL, 1);
+
+INSERT INTO `schritte` (`SchritteID`, `Titel`, `Prioritaet`, `Reihenfolge`, `Bearbeitungsstatus`, `BearbeitungsZeitpunkt`, `BearbeitungsBeginn`, `BearbeitungsEnde`, `homeverzeichnisNutzen`, `typMetadaten`, `typAutomatisch`, `typImportFileUpload`, `typExportRus`, `typImagesLesen`, `typImagesSchreiben`, `typExportDMS`, `typBeimAnnehmenModul`, `typBeimAnnehmenAbschliessen`, `typBeimAnnehmenModulUndAbschliessen`, `typAutomatischScriptpfad`, `typBeimAbschliessenVerifizieren`, `typModulName`, `BearbeitungsBenutzerID`, `ProzesseID`, `edittype`, `scriptName1`, `scriptName2`, `typAutomatischScriptpfad2`, `scriptName3`, `typAutomatischScriptpfad3`, `scriptName4`, `typAutomatischScriptpfad4`, `scriptName5`, `typAutomatischScriptpfad5`, `typScriptStep`, `batchStep`, `stepPlugin`, `validationPlugin`) VALUES
+(3, 'Structure and metadata', 0, 3, 0, '2016-02-03 16:49:48', NULL, NULL, 0, b'1', b'0', b'0', b'0', b'0', b'0', b'0', b'1', b'0', b'0', '', b'0', NULL, 1, 1, 4, '', '', '', '', '', '', '', '', '', b'0', b'0', '', ''),
+(1, 'Scanning', 0, 1, 1, '2016-03-02 11:38:28', NULL, NULL, 0, b'0', b'0', b'0', b'0', b'1', b'1', b'0', b'0', b'0', b'0', '', b'0', NULL, 1, 1, 4, '', '', '', '', '', '', '', '', '', b'0', b'0', '', ''),
+(2, 'QC', 0, 2, 0, '2016-02-03 16:49:48', NULL, NULL, 0, b'0', b'0', b'0', b'0', b'1', b'0', b'0', b'0', b'0', b'0', '', b'0', NULL, 1, 1, 4, '', '', '', '', '', '', '', '', '', b'0', b'0', '', ''),
+(4, 'Export DMS', 0, 4, 0, '2016-02-03 16:49:48', NULL, NULL, 0, b'0', b'0', b'0', b'0', b'0', b'0', b'1', b'0', b'0', b'0', '', b'0', NULL, 1, 1, 4, '', '', '', '', '', '', '', '', '', b'0', b'0', '', '');
+
+INSERT INTO `schritteberechtigtegruppen` (`BenutzerGruppenID`, `schritteID`) VALUES
+(1, 1),
+(1, 2),
+(1, 3),
+(1, 4),
+(2, 1),
+(3, 2),
+(5, 3),
+(6, 4);
```

Comment:
This new file would fail several submission tests (lines ending with whitespace, inconsistent indentation, missing newline at last line).